### PR TITLE
Fix: Auth refetch

### PR DESCRIPTION
**Pull request by rasmusguterstam**
# Jira Issue: [IP-34](https://hedvig.atlassian.net/browse/IP-34)

## What & Why
- Add "Not Found" page for unmapped routes
- Fix authentication bug where it doesn't refetch on a 403, even though it gets a new access token
- Fix minor routing bug using hot-reload (was only an issue when running locally)

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [x] Codescouted
- [ ] Unit tests written
- [x] Tested locally

### CODE REVIEW

**Comment by fd-pliska** [comment]
I have very little understanding of what's going on, but it looks nice!

**Comment by maxplt** [comment]
Looks perfect 🔥

**Review by vonElfvin — APPROVED** [review]
Looks really good! 💯  Feels like there should be some way to make this even simpler by talking directly to gatekeeper but I cannot figure anything out right now 🤔

File: src/App.tsx [hunk 1]
```diff
@@ -116,21 +116,37 @@ const App: React.FC = () => {
...
+                            <Route
+                              component={() => (
+                                <StandaloneMessage paddingTop={'25vh'}>
+                                  Page not found
+                                </StandaloneMessage>
+                              )}
+                            />
+                          </Switch>
+                        )}
                       </Switch>
```
[the remark this answers is not in the document]
Comment (rasmusguterstam) [reply src/App.tsx:150]: Apparently it is required, but I'm not really sure why. My qualified guess is that it doesn't know how to behave and begins to act up when we conditionally drop some new routes into the old switch!

**Pull request merged**